presto-docs/README.md

```diff
@@ -0,0 +1,73 @@
+# presto-docs - Presto Documentation
+
+The presto-docs module contains the reference documentation for Presto.
+
+## Tools
+
+The engine used to create the documentation in HTML format is the Python-based
+[Sphinx](https://www.sphinx-doc.org). Installation instructions for various OS
+are [available on the Sphinx site](https://www.sphinx-doc.org/en/master/usage/installation.html).
+
+Documentation source files can be found in [Restructured
+Text](https://en.wikipedia.org/wiki/ReStructuredText) (`.rst`) format in
+`src/main/sphinx` and sub-folders.
+
+In addition, `make` is required
+
+## Build and Read
+
+With the tools installed and available on the PATH, you can build the docs
+easily with make:
+
+```bash
+cd presto/presto-docs
+make clean html
+```
+
+Alternatively you can keep using Apache Maven and Java like for the rest of the
+Presto build. You just need to have built the current version from the root.
+Subsequently you can build the site:
+
+```bash
+cd presto/presto-docs
+mvn clean install
+```
+
+It calls the make command and performs other checks, so you still need the tools
+installed.
```

Comment:
Do you? I thought maven sphinx plugin will use a bundled version.
I think this is how it works on Travis.
(I can't verify locally because, well, I already have the tools installed.)

Reply:
I just checked the source of the maven plugin. It has a sphinx version embedded it seems ... I assumed otherwise. Seems like a bad idea to me.. it still needs python for example. Anyway .. I updated this paragraph.